demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2962,6 +2997,17 @@ export class DemoMeetingApp
       return new ResizeProcessor(0.5625);  // 16/9 Aspect Ratio
     }
 
+    if (videoFilter === 'BackgroundBlur') {
+      console.log("background blur - create called from videoFilterToProcessor!")
+      const bbprocessor = await BackgroundBlurVideoFrameProcessor.create(this.getBackgroundBlurSpec());
+      bbprocessor.addObserver({
```

Comment:
Should we make this a named or referenced observer function so that we can remove it while leaving or cleaning up resources on meeting leave?

https://github.com/aws/amazon-chime-sdk-js/blob/cb085eeac667398101ca2b30c909898cf590ddd9/demos/browser/app/meetingV2/meetingV2.ts#L3185-L3236

Reply:
added the named observer and removed at the end